docs/understanding-airbyte/basic-normalization.md

```diff
@@ -1,5 +1,11 @@
 # Basic Normalization
 
+:::danger
+
+Basic normalization is being removed in favor of [Typing and Deduping](/understanding-airbyte/typing-deduping), as part of [Destinations V2](/release_notes/upgrading_to_destinations_v2). This pages remains as a guide for legacy connectors.
```

Comment:
do we want to specify which connectors are already on dv2 and which are still upcoming?

Reply:
If we do, it should be on one of the linked pages rather than this one